db/change_cache_test.go

```diff
@@ -327,8 +321,8 @@ func TestChannelCacheBackfill(t *testing.T) {
 		t.Skip("This test does not work with XATTRs due to calling WriteDirect().  Skipping.")
 	}
 
-	base.EnableLogKey("Cache")
-	base.EnableLogKey("Changes+")
+	base.ConsoleLogLevel().Set(base.LevelDebug)
+	base.ConsoleLogKey().Set(base.KeyCache | base.KeyChanges)
```

Comment:
Can we leave these as EnableLogKey, but change EnableLogKey to enable Debug (if it sees a trailing +), and then set the Log Key?

Reply:
Good idea to switch EnableLogKey to EnableLogKeyForTest (and maybe even move to utilities_testing)